packages/react-aria-components/src/ToggleButton.tsx

```diff
@@ -19,7 +19,7 @@ import {ToggleState, useToggleState} from 'react-stately';
 export interface ToggleButtonRenderProps extends ButtonRenderProps {
   /**
    * Whether the button is currently selected.
-   * @selector [aria-pressed=true]
+   * @selector data-pressed=true]
```

Comment:
I think `data-pressed` is already used for the mouse down state (extended from ButtonRenderProps). We should call this one `data-selected` to match the prop name.